processor/framework.py

```diff
@@ -324,31 +320,21 @@ def htcondor_job_config(self, config, job_num, branches):
         )
         for file_ in ["Log", "Output", "Error"]:
             os.makedirs(os.path.join(logdir, file_), exist_ok=True)
-        logfile = os.path.join(
-            logdir, "Log", "{}_{}to{}.txt".format(task_name, branches[0], branches[-1])
-        )
-        outfile = os.path.join(
-            logdir,
-            "Output",
-            "{}_{}to{}.txt".format(task_name, branches[0], branches[-1]),
-        )
-        errfile = os.path.join(
-            logdir,
-            "Error",
-            "{}_{}to{}.txt".format(task_name, branches[0], branches[-1]),
-        )
+        logfile = os.path.join(logdir, "Log", task_name)
```

Comment:
please add .txt as a file ending to the logfiles ( I know not needed technically but makes it easier for people to understand)